### PR TITLE
[Injectors docs ] withFailure docs

### PR DESCRIPTION
**Pull request by DLotero**
Created documentation for the withFailure injector

Trello card : https://trello.com/c/GLT020M6/40-injectors-docs

### CODE REVIEW

**Comment by damfinkel** [comment]
Please squash the commits if you can since they are called the same and the second one is a bugfix of the first. Do it after the PR is approved by all assignees please :)

File: src/injections/withFailure/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
+
```
Comment (marianozicavo) [inline src/injections/withFailure/README.md]: Can you delete this empty lines?

File: src/injections/withFailure/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  import { withFailure } from 'redux-recompose';
+
+  const actionCreators = {
+    someAction: data => ({
+      type: actionType,
+      target: someTarget,
+      service: someService,
+      payload: data,
+      injections: [
+        withFailure((dispatch, response) => {
```
Comment (marianozicavo) [inline src/injections/withFailure/README.md]: I'd add state `(dispatch, response, state)`

File: src/injections/withFailure/README.md (hunk 1 above)
Comment (mvbattan) [reply src/injections/withFailure/README.md]: Would you like to *

**Pull request merged**